src/psyclone/tests/domain/gocean/transformations/gocean_opencl_trans_test.py

```diff
@@ -341,6 +341,8 @@ def test_invoke_opencl_initialisation_grid():
       integer(kind=c_intptr_t) cl_mem
       integer ierr
 
+      ! psyclone codeblock reason:
+      !  - unsupported statement
       cmd_queues => get_cmd_queues()
```

Comment:
It would be nice if we could also mark the end of a CodeBlock. How easy/difficult would that be?

Reply:
We would need a new property in CommentableMixin so that it is attached with the right node and a bit of infrastructure in the backend visitor similar to the preceding_comment.

However, most of the CodeBlocks are only one statement and it adds lots of visual clutter when adding end comments. When there are more than one the one reason item per statement also makes it quite clear. So in my subjective opinion the preceding_comment-only looks better, but let me know if you still disagree.

Reply:
I like the type information we have now and am inclined to agree with you so let's leave it.